doc/source/tutorial/linalg_batch.md

```diff
@@ -0,0 +1,148 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
```

Comment:
This is particular to this page, because it hasn't been added to any toctree. If it is added to a regular toctree, we don't need this option. But it is informational in that there are a number of options that can be added to the MyST frontmatter - see https://myst-nb.readthedocs.io/en/latest/authoring/text-notebooks.html#notebook-level-metadata

Reply:
Thanks!